homeassistant/components/gios/air_quality.py

```diff
@@ -80,7 +80,7 @@ def icon(self) -> str:
     @property
     def air_quality_index(self) -> str | None:
         """Return the air quality index."""
-        return cast(Optional[str], self.coordinator.data.get(API_AQI, {}).get("value"))
+        return cast(Optional[str], self.coordinator.data.get(API_AQI).get("value"))
```

Comment:
Why did we change this line? If the first dict lookup fails to find the key the second lookup will error.

Reply:
Yes, that was my stupid mistake, this line should look like this:
```python
return cast(Optional[str], self.coordinator.data[API_AQI].get("value"))
```
GIOS `air_quality` platform has been removed, so we don't use that code anymore.

Reply:
Ok, good!